### DISCUSSION TITLE
Extra filter types

### DISCUSSION THREAD

**Discussion by samgranger** [rappasoft/laravel-livewire-tables#404] [Feature Requests]
It would be nice to see the following types of filters:

- Numeric range, so if you have a column with numeric values, users can specify a number between x and y.
- Multiselect, so users can filter where column value equals x or y.

**Comment by rappasoft** [comment]
I don't know if I have time this month to do it, but you can always submit a PR and I'll merge or build on it. You could use this one as a reference: https://github.com/rappasoft/laravel-livewire-tables/pull/332

Right now I don't have any support for side-by-side filters which I've been meaning to add. Which would probably be needed for a range from a UI perspective. Right now my "from - to" date ranges in my apps are just on top of each other.

**Comment by AlexHaible** [comment]
Adding to this: A simple string filter would also be great to have.
